cmd/nfs-subdir-external-provisioner/provisioner.go

```diff
@@ -139,8 +139,8 @@ func (p *nfsProvisioner) Provision(ctx context.Context, options controller.Provi
 
 func (p *nfsProvisioner) Delete(ctx context.Context, volume *v1.PersistentVolume) error {
 	path := volume.Spec.PersistentVolumeSource.NFS.Path
-	relativePath := strings.Replace(path, p.path, "", 1)
-	oldPath := filepath.Join(mountPath, relativePath)
+	basePath := filepath.Base(path)
+	oldPath := filepath.Join(mountPath, basePath)
```

Comment:
To be sure, the end result is the same as before the PR, just using `filepath.Base`, right?

Reply:
The result is definitelly not the same :-). Archived directory has the same name as the provisioned directory, just with 'archived-' prefix. And that is the same behavior that the old nfs provisioner had. It is much more user friendly, than just plain guid PV name used now.

Reply:
Oh...now I know what you meant with the question. Yes, the `oldPath` will resolve to the same value.